superset/connectors/sqla/models.py

```diff
@@ -535,16 +542,25 @@ def _process_sql_expression(
 
 
 class SqlaTable(Model, BaseDatasource):  # pylint: disable=too-many-public-methods
-    """An ORM object for SqlAlchemy table references"""
+    """An ORM object for SqlAlchemy table references."""
 
     type = "table"
     query_language = "sql"
     is_rls_supported = True
-    columns: List[TableColumn] = []
-    metrics: List[SqlMetric] = []
+    columns: Mapped[List[TableColumn]] = relationship(
```

Comment:
Ideally we would use `lazy="selectin"` rather than `lazy="select"` (default) but this seems to be causing downstream issues with the associated `sqlatable_user` table. My current hunch is the `selectin` option is rather new and there's been a number of [reported](https://github.com/sqlalchemy/sqlalchemy/issues?q=is%3Aissue+selectin+is%3Aclosed) issues.